AEPServices/Sources/ui/fullscreen/FullscreenMessaging.swift

```diff
@@ -12,20 +12,18 @@
 
 import Foundation
 
-/// Fullscreen message event listener
-public protocol FullscreenListenerInterface {
-
+/// Fullscreen message lifecycle event listener
+@objc(AEPFullscreenMessaging) public protocol FullscreenMessaging {
```

Comment:
Any reason we remove the `message` parameter from these functions? I feel it could be helpful.